src/pages/billing/List/ProjectListItem.ts

```diff
@@ -17,66 +17,95 @@ import { BillingProjectActions } from 'src/pages/billing/List/BillingProjectActi
+  isDeleting: boolean
 }
 
 export const ProjectListItem = (props: ProjectListItemProps) => {
+  const [hovered, setHovered] = useState<boolean>()
+
   const { projectName, roles, status, message, cloudPlatform } = props.project
+  const viewerRoles = _.intersection(roles, _.values(billingRoles))
+  const isOwner = _.includes(billingRoles.owner, roles)
+
   // Billing projects in an error status may have UNKNOWN for the cloudPlatform.
-  const cloudContextIcon = isKnownCloudProvider(cloudPlatform) && div({ style: { display: 'flex', marginRight: '0.5rem' } }, [
+  const cloudContextIcon = isKnownCloudProvider(cloudPlatform) && span({ style: { marginRight: '0.5rem' } }, [
     h(CloudProviderIcon, { cloudProvider: cloudPlatform })
   ])
 
-  const selectableProject = () => h(Clickable, {
-    style: { ...styles.projectListItem(props.isActive), color: props.isActive ? colors.accent(1.1) : colors.accent() },
+  const projectNameElement = span({ style: { wordBreak: 'break-all', verticalAlign: 'text-top' } }, [projectName])
+
+  const actionElement = isOwner && h(BillingProjectActions, { projectName: props.project.projectName, loadProjects: props.loadProjects })
+
+  const renderSelectableProject = () => div({
+    style: { ...listItemStyle(props.isActive, hovered) },
+    onMouseEnter: () => setHovered(true),
+    onMouseLeave: () => setHovered(false)
+  }, [h(Clickable, {
+    style: { color: props.isActive ? colors.accent(1.1) : colors.accent() },
     href: `${Nav.getLink('billing')}?${qs.stringify({ selectedName: projectName, type: 'project' })}`,
     onClick: () => Ajax().Metrics.captureEvent(Events.billingProjectOpenFromList, extractBillingDetails(props.project)),
-    hover: Style.navList.itemHover(props.isActive),
     'aria-current': props.isActive ? 'location' : false
-  }, [cloudContextIcon, projectName])
+  }, [cloudContextIcon, projectNameElement]), actionElement])
 
-  const unselectableProject = () => {
-    const iconAndTooltip =
-      // @ts-ignore
-      props.isCreatingStatus ? spinner({ size: 16, style: { color: colors.accent(), margin: '0 1rem 0 0.5rem' } }) :
-        status === 'Error' ? h(Fragment, [
-          // @ts-ignore
-          h(InfoBox, { style: { color: colors.danger(), margin: '0 0.5rem 0 0.5rem' }, side: 'right' }, [
-            div({ style: { wordWrap: 'break-word', whiteSpace: 'pre-wrap' } }, [
-              message || 'Error during project creation.'
-            ])
-          ]),
-          //Currently, only billing projects that failed to create can have actions performed on them.
-          //If that changes in the future, this should be moved elsewhere
-          isOwner && h(BillingProjectActions, { projectName: props.project.projectName, loadProjects: props.loadProjects })
-        ]) : undefined
+  const renderUnselectableProject = () => {
+    const isCreatingOrDeleting = props.isCreating || props.isDeleting
+    const isErrorStatus = _.includes(status, ['Error', 'DeletionFailed'])
```

Comment:
I don't know how to get a billing project into the "DeletionFailed" status, so hoping that my unit test closely mimics reality.

Reply:
Actually, I hit it!

<img width="535" alt="image" src="https://user-images.githubusercontent.com/484484/225772153-119cc8aa-94b9-4f31-897b-6c7439e84228.png">

<img width="633" alt="image" src="https://user-images.githubusercontent.com/484484/225772186-c9904928-25a9-4f31-9d09-909a0ef23bd6.png">

Reply:
@blakery I'm not having any luck deleting this billing project, and I wonder if it will impact all billing projects that used to create an Azure workspace. This is what I did:

1. Created a billing project co-owned by another user
2. Logged in as that user and created a workspace
3. Deleted the workspace (which took a long time due to WDS)
4. Tried to delete the billing project, got error
5. Tried to delete the billing project again, got same error

Reply:
Actually it's not true that it would block billing project deletion-- the damage is done at the time the workspace itself is deleted.

Reply:
Yeah, I thought we'd done that as well as the linked tickets to make BPM and WSM resilient to it so I was mostly afraid we'd had a regression

Reply:
Posted https://broadinstitute.slack.com/archives/CA3NP1733/p1679067806962069

Reply:
Turns out to be a known issue: https://broadworkbench.atlassian.net/browse/IA-4097